org.sonarlint.eclipse.core.tests/src/test/java/org/sonarlint/eclipse/core/internal/jobs/AnalyzeStandaloneProjectJobTest.java

```diff
@@ -62,16 +60,6 @@ public class AnalyzeStandaloneProjectJobTest extends SonarTestCase {
   private static LogListener listener;
   private static IProject project;
 
-  @BeforeClass
```

Comment:
In fact it should be done already, since the backend is started in the bundle activator. The issue previously was that we were starting multiple backends...

Reply:
Ah, I was thinking that might be one of the only reasons, makes sense!